src/forms/brownbag-submissions.njk

```diff
@@ -56,42 +56,300 @@ permalink: brownbag-idea/
           <small id="emailHelp" class="form-text text-muted">Required. We'll never share your email with anyone else.</small>
         </div>
       </fieldset>
+
       <fieldset>
-        <legend>Your Brownbag Idea:</legend>
+        <legend>Your Brownbag Idea</legend>
+        <p class="text-muted">This section is to gather some general information about the talk you're proposing. Don't worry if you don't have all the details ironed out yet—we can work with you to finalize your talk's title and description!</p>
         <div class="form-group">
-          <label for="topicTitle">Title of the Brownbag</label>
+          <label for="topicTitle">What is the proposed title of your talk?</label>
           <input
             type="text"
             class="form-control"
             id="topicTitle"
             name="topicTitle"
-            required
           />
         </div>
         <div class="form-group">
-          <label for="topicIdea">Description we can share on the event page.</label>
-          <textarea class="form-control" required id="topicIdea" name="topicIdea" rows="3"></textarea>
+          <label for="topicDescription">What is the description of your talk?</label>
+          <textarea
+            class="form-control"
+            id="topicDescription"
+            name="topicDescription"
+            rows="3">
+          </textarea>
+        </div>
+        <p>Please give three dates and times you're available to present in order of preference. Be sure to include timezone!</p>
```

Comment:
Looking at @BekahHW's comments, can we maybe change this to have these three options with a disclaimer that we will not do brownbags on Tuesday and Thursdays due to those being our coffee days:
1. Friday at 12:00 PM ET (the de-facto brownbag time)
2. Monday at 9:00 AM ET (the non-Noon set brownbag time)
3. Other